tests/trainers/test_change.py

```diff
@@ -341,3 +341,171 @@ def test_multiclass_predict(self, fast_dev_run: bool) -> None:
             max_epochs=1,
         )
         trainer.predict(model=model, datamodule=datamodule)
+
+    def test_levircd_temporal_correspondence(self) -> None:
+        """Test LEVIR-CD temporal correspondence issues from GitHub issue #2920."""
+        pytest.importorskip(
+            'torchgeo.datamodules.levircd', reason='LEVIRCDDataModule required'
+        )
+
+        from torchgeo.datamodules.levircd import LEVIRCDDataModule
```

Comment:
Why would this ever not be importable? I think you can skip this and import at the top.

I appreciate the thorough testing, but I do wonder if it's too much. Perhaps we could instead unit test the transform? Or move the tests to `tests/datamodules/test_levircd.py`? I'm trying to slowly get rid of those tests though. I also eventually want to move this transform to Kornia so we don't have to maintain it anymore.